src/50_Sample_AMPs/Recipe.html

```diff
@@ -0,0 +1,466 @@
+      padding-left: 72px;
+      box-shadow: 0 2px 2px 0 rgba(0,0,0,.14),0 3px 1px -2px rgba(0,0,0,.2),0 1px 5px 0 rgba(0,0,0,.12);
+    }
+    .logo > a {
+      font-size: 16px;
+      font-weight: 500;
+      color: white;
+      text-transform: uppercase;
+    }
+    .heading {
+      padding-bottom: 8px;
+    }
+    .heading > #summary {
+      font-weight: 500;
+    }
+    .heading > small {
+      color: #656565;
+    }
+    .related {
+      background-color: #f5f5f5;
+      margin: 16px 16px;
+      display: block;
+      color: #111;
+      height: 75px;
+      padding: 0;
+    }
+    .related > span {
+      font-size: 16px;
+      line-height: 75px;
+      font-weight: 400;
+      vertical-align: top;
+      margin: 8px;
+    }
+    .related:hover {
+      background-color: #ccc;
+    }
+  </style>
+
+  <!-- Load the AMP library from the CDN. -->
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+
+</head>
+<body>
+
+  <!-- -->
+  <div class="logo">
+    <a href="/"> AMP by Example</a>
+  </div>
+
+  <!-- #### Basic Usage -->
+  <!--
+  We'll start the recipe article with its title, followed my a mouthwatering gallery of shots taken during preparation.
+  -->
+  <div class="heading">
+    <h1>Garlic-studded roast lamb</h1>
+    <p id="summary">This is a sample recipe AMP article demonstrating how to express machine-readable recipe data using JSON+LD. It uses AMP components that work well for recipe articles: image gallery (carousel), reviews carousel, social sharing, and related articles. There are also star ratings, read-only for now (see <a href="https://github.com/ampproject/amphtml/issues/2691">this GitHub issue</a>).</p>
+    <p><small>By Dan Dascalescu<br>
+      Published: June 12, 2016</p></small></p>
+  </div>
+
+  <!-- #### Social Sharing  -->
+
+  <!--
+  The Social Share extension provides a common interface for share buttons.  Learn more about [`amp-social-share`](/components/amp-social-share/).
+  -->
+  <p class="heading">
+    <amp-social-share type="twitter" data-param-text="Amazing recipe that my friend made" width="45" height="33"></amp-social-share>
+    <amp-social-share type="facebook" width="45" height="33" data-attribution=254325784911610></amp-social-share>
+    <amp-social-share type="gplus" width="45" height="33"></amp-social-share>
+    <amp-social-share type="email" width="45" height="33"></amp-social-share>
+    <amp-social-share type="pinterest" width="45" height="33"></amp-social-share>
+  </p>
+
+  <!-- #### Image Galleries  -->
+  <!--
+  Use `amp-carousel` for image galleries. Here we display responsively images with different aspect ratios. Learn more about [creating image galleries in AMP](/advanced/image_galleries_with_amp-carousel/). The `loop` attribute lets us start with a picture of the finished product, and get to the same finished when advancing to the slide after the picture of the dish just before putting it in the oven. What You Start With Is What You Get.
```

Comment:
That the first slide in the carousel was also the final result, since it loops. Removed that entirely, and [moved the "ready" slide to a hero image](https://github.com/ampproject/amp-by-example/pull/192#discussion_r66869133).